crates/sui-framework/packages/deepbook/sources/clob.move

```diff
@@ -410,22 +408,19 @@ module deepbook::clob {
                     // Total quote quantity paid by taker.
                     // filled_quote_quantity_without_commission * (FLOAT_SCALING + taker_fee_rate) = filled_quote_quantity
                     let filled_quote_quantity_without_commission: u64;
-                    if (taker_quote_quantity_remaining >= maker_quote_quantity) {
+                    if (taker_quote_quantity_remaining > maker_quote_quantity) {
```

Comment:
Is there a test that fails if this is `>=` rather than `>`? Might be good to isolate this and other semantic changes/bugfixes in a separate PR

Reply:
The change here is to make the code concise and easier to understand. We change ">=" to ">"  to delete the if judgment here. (Line 458 -Line 460 )

https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/packages/deepbook/sources/clob.move#L458
 if (taker_quote_quantity_remaining == 0) {
        terminate_loop = true;
};

Reply:
@sblackshear `>=` and `>` are equivalent in this case since when `taker_quote_quantity_remaining  == maker_quote_quantity`, `filled_quote_quantity = taker_quote_quantity_remaining = maker_quote_quantity`

Reply:
Makes sense. Would you mind separating this change into its own PR that explains this in the summary, rather than grouping it with these unrelated changes?

Reply:
@sblackshear Actually, the intention of this change is to make the test coverage happy. So maybe this change should fit into this PR rather than a separate PR?